graphql/error.js

```diff
@@ -0,0 +1,9 @@
+import { builtinModules } from "module";
+
+class Error {
+  constructor(message) {
+    this.message = message
+  }
+}
+
+modules.export = Error
```

Comment:
No new line?!

Reply:
anarchy!

Reply:
also, should be module.exports not modules.export.